src/client/game/board.ts

```diff
@@ -261,6 +274,10 @@ export class Board {
       subheading = "";
     }
     const gameStatus = document.getElementById("gameStatus");
+    if (!gameStatus) {
+      showGameError(new ErrGeneric("Game status DOM element not found"));
```

Comment:
showGameError() is a new method to surface errors to the client within game UI. I will highlight its implementation as a separate comment below.